storage/tokudb/ha_tokudb_alter_56.cc

```diff
@@ -248,6 +248,40 @@ static bool only_flags(ulong bits, ulong mask) {
     return (bits & mask) != 0 && (bits & ~mask) == 0;
 }
 
+// Table create options that should be ignored by TokuDB
+// There are 25 total create options defined by mysql server(see handler.h),
```

Comment:
Still no space between "server" and "(", but, meh :)

Reply:
Gah, sorry about that, I (obviously) missed that part of your comment. It's not a problem, I'll fix it and re-push after a local mtr cycle, then merge without needing more review.